modules/final_radiological_review/php/NDB_Menu_Filter_final_radiological_review.class.inc

```diff
@@ -416,6 +416,7 @@ class NDB_Menu_Filter_Final_Radiological_Review extends NDB_Menu_Filter
         return array_merge(
             $deps,
             array(
+             $baseURL . "/final_radiological_review/js/columnFormatter_HistoryLog.js",
```

Comment:
This needs to go in `NDB_Form_final_radiological_review.class.inc`. You'll also need to add 
```
$baseURL . '/js/components/StaticDataTable.js',
$baseURL . '/js/components/DynamicDataTable.js',
```